web-ui/src/backup_account/backup_email/backup_email.js

```diff
@@ -72,12 +74,10 @@ export class BackupEmail extends React.Component {
           <p>{t('backup-account.backup-email.paragraph2')}</p>
           <InputField name='email' label={t('backup-account.backup-email.input-label')} errorText={this.state.error} onChange={this.validateEmail} />
           <SubmitButton buttonText={t('backup-account.backup-email.button')} disabled={this.state.submitButtonDisabled} />
-          <div className='link-content'>
-            <a href='/' className='link'>
-              <i className='fa fa-angle-left' aria-hidden='true' />
-              <span>{t('back-to-inbox')}</span>
-            </a>
-          </div>
+          <BackLink
+            href='/'
+            text={t('back-to-inbox')}
+          />
```

Comment:
Should we also this component on the 'Confirmation' page of the backup-account flow?